apps/bare-expo/android/app/build.gradle

```
@@ -135,6 +135,15 @@ android {
     androidResources {
         ignoreAssetsPattern '!.svn:!.git:!.ds_store:!*.scc:!CVS:!thumbs.db:!picasa.ini:!*~'
     }
+    if (project.properties["expo.inlineModules.enabled"] == "true") {
```

```suggestion
    if (findProperty('expo.inlineModules.enabled') == 'true') {
```

findProperty can search up to root project